fixfasta.py

```
@@ -399,15 +398,14 @@ def main():
     
     try:
         # Setup input/output
-        if args.files:
-            input_stream = fileinput.input(args.files)
-        else:
-            input_stream = sys.stdin
+        input_stream = fileinput.input(args.files) if args.files else sys.stdin
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)